ntbs-service/Pages/Notifications/Edit/ClinicalDetails.cshtml

```diff
@@ -329,42 +311,36 @@
                                 Patient started treatment
                             </label>
                         </div>
-                        <validate-date ref="date3" name="Treatment Start Date" model="ClinicalDetails" property="TreatmentStartDate" rank="3" v-model="dates[3]" inline-template>
-                            <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-treatment-conditional">
-                                @{
-                                    var TreatmentError = !Model.IsValid("ClinicalDetails.TreatmentStartDate");
-                                    var TreatmentGroupState = TreatmentError ? Error : Standard;
-                                }
+                        <div class="govuk-radios__conditional govuk-radios__conditional--hidden" id="conditional-treatment-conditional">
+                            @{
+                                var TreatmentError = !Model.IsValid("ClinicalDetails.TreatmentStartDate");
+                                var TreatmentGroupState = TreatmentError ? Error : Standard;
+                            }
+                            <validate-date ref="date3" name="Treatment Start Date" model="ClinicalDetails" property="TreatmentStartDate" rank="3" v-model="dates[3]" inline-template>
```

Comment:
Is `validate-date` required to be the immediate parent of the `nhs-form-group` elements to function correctly? If so, then we should make it explicit, at least with a comment on the component (and perhaps we should make this dependency more concrete (ref?), but I don't think this is the place to fight that battle ;) )

Reply:
I will create a tech debt ticket to go over all form groups are ref those used in validation